packages/devextreme/scss/widgets/base/splitter/_index.scss

```diff
@@ -14,5 +15,6 @@
 
 .dx-resize-handle {
   cursor: col-resize;
-  flex-basis: 1px;
+  flex-basis: 8px;
+  border: 1px solid gray;
```

Comment:
Better will use background color for this purpose. Color for draft is #D1D1D1.

Reply:
Let's stretch resizeHandle of full width, height for corresponding orientations.

Reply:
Also will better if pointer types for horizontal, vertical resizeHandles will right